pwnlib/tubes/process.py

```diff
@@ -751,7 +804,7 @@ def close(self):
                 try:
                     fd.close()
                 except IOError as e:
-                    if e.errno != errno.EPIPE:
+                    if e.errno != errno.EPIPE and e.errno != errno.EINVAL:
```

Comment:
EINVAL? how? :D

Reply:
That's returned when the fd was already closed somewhere.